keyboards/dztech/dz65rgb/keymaps/drootz/keymap.c

```diff
@@ -0,0 +1,774 @@
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, 
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, 
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, 
+        KC_TRNS, KC_LALT, KC_LGUI, KC_TRNS, KC_TRNS, MO(_FN), KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS
+    ),
+    [_FN] = LAYOUT_65_ansi(
+        DYN_REC_STOP, KC_F1, KC_F2, KC_F3, KC_F4, KC_F5, KC_F6, KC_F7, KC_F8, KC_F9, KC_F10, KC_F11, KC_F12, KC_NO, KC_NO, 
+        KC_TRNS, KC_MUTE, KC_VOLU, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, RGB_SAD, RGB_SAI, DYN_MACRO_PLAY2, DYN_REC_START2, 
+        KC_TRNS, KC_BRID, KC_VOLD, KC_BRIU, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, RGB_HUD, RGB_HUI, DYN_MACRO_PLAY1, DYN_REC_START1, 
+        KC_TRNS, TO(_MAIN), TO(_MAC), KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, RGB_SPD, RGB_SPI, KC_TRNS, RGB_VAI, KC_NO, 
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, RGB_RMOD, RGB_VAD, RGB_MOD
+    ),
+}; 
+
+//**************** MATRIX SCANS *********************//
+
+void rgb_matrix_indicators_user(void) { 
+
+    #ifdef RGB_MATRIX_ENABLE
+
+    HSV hsvConfig = {rgb_matrix_config.hsv.h, rgb_matrix_config.hsv.s, rgb_matrix_config.hsv.v};
+    RGB rgbConfig = hsv_to_rgb(hsvConfig);
+    const uint8_t CONFIG_HSV[3] = {rgbConfig.r, rgbConfig.g, rgbConfig.b};
+
+    HSV hsvConfigDim = {rgb_matrix_config.hsv.h, rgb_matrix_config.hsv.s, rgb_matrix_config.hsv.v * ledDimRatio};
+    RGB rgbConfigDim = hsv_to_rgb(hsvConfigDim);
+    const uint8_t CONFIG_HSV_DIM[3] = {rgbConfigDim.r, rgbConfigDim.g, rgbConfigDim.b};
+
+    HSV hsvWhiteDim = {rgb_matrix_config.hsv.h, 0, rgb_matrix_config.hsv.v * ledDimRatio};
+    RGB rgbWhiteDim = hsv_to_rgb(hsvWhiteDim);
+    const uint8_t WHITE_HSV_DIM[3] = {rgbWhiteDim.r, rgbWhiteDim.g, rgbWhiteDim.b};
+
+    HSV hsvRed = {0, rgb_matrix_config.hsv.s, rgb_matrix_config.hsv.v};
+    RGB rgbRed = hsv_to_rgb(hsvRed);
+    const uint8_t RED_HSV[3] = {rgbRed.r, rgbRed.g, rgbRed.b};
+
+    HSV hsvRedDim = {0, rgb_matrix_config.hsv.s, rgb_matrix_config.hsv.v * ledDimRatio};
+    RGB rgbRedDim = hsv_to_rgb(hsvRedDim);
+    const uint8_t RED_HSV_DIM[3] = {rgbRedDim.r, rgbRedDim.g, rgbRedDim.b};
+
+    HSV hsvGreenDim = {85, rgb_matrix_config.hsv.s, rgb_matrix_config.hsv.v * ledDimRatio};
+    RGB rgbGreenDim = hsv_to_rgb(hsvGreenDim);
+    const uint8_t GREEN_HSV_DIM[3] = {rgbGreenDim.r, rgbGreenDim.g, rgbGreenDim.b};
+
+    /* CapsLock LED indicator */
+    if (IS_HOST_LED_ON(USB_LED_CAPS_LOCK)) {
+        rgb_matrix_set_color(30, 0xFF, 0xFF, 0xFF);
+    } 
+
+    /* Current layer LED indicator */
+    rgb_matrix_set_color(layers_leds_map[biton32(layer_state)], WHITE_HSV_DIM[0], WHITE_HSV_DIM[1], WHITE_HSV_DIM[2]); 
```

Comment:
Best to use the typedef for the layer check.
```suggestion
    rgb_matrix_set_color(layers_leds_map[get_highest_layer(layer_state)], WHITE_HSV_DIM[0], WHITE_HSV_DIM[1], WHITE_HSV_DIM[2]); 
```

Reply:
Thanks! Addressed in PR #8199 Feedback Commit #1 - c0be53d